doc/service_example.gv

```diff
@@ -0,0 +1,22 @@
+digraph {
+    rankdir="LR";
+    service_in -> node_domain_1 [dir="both"] [label="cli"]
```

Comment:
```suggestion
    service_in -> node_domain_1 [dir="both"] [label="client"]
```

So it's not confused with "command line interface".

Reply:
Fixed in https://github.com/ros2/domain_bridge/pull/26/commits/a549c594120b6c77d35d67b022c29d45ff1069a7